src/services/initializer-service.ts

```diff
@@ -54,6 +61,7 @@ export class InitializerService {
     fs.mkdirSync(this.filesys.getBundleFilePath('microservices'))
     fs.mkdirSync(this.filesys.getBundleFilePath('microfrontends'))
     fs.mkdirSync(this.filesys.getBundleFilePath('epc'))
+    fs.mkdirSync(this.filesys.getBundleFilePath('aux'))
```

Comment:
You can use the constants defined in `paths` here

Reply:
done